eng/pipelines/common/templates/runtimes/run-test-job.yml

```diff
@@ -64,6 +64,8 @@ jobs:
         - '${{ parameters.runtimeFlavor }}_common_test_build_p1_AnyOS_AnyCPU_${{parameters.buildConfig }}'
       - ${{ if ne(parameters.stagedBuild, true) }}:
         - ${{ if or( eq(parameters.runtimeVariant, 'minijit'), eq(parameters.runtimeVariant, 'monointerpreter'))  }}:
+          # This is needed for creating a CORE_ROOT in the current design.
```

Comment:
courious, shouldn't this be when `runtimeFlavor` is mono? that way we cover all mono flavors? Or are `minjit`/`monointerpreter` always a value for when we are running tests on mono?

Reply:
It's the ones where I saw the issue, I thought about it, but I am not sure how the other mono runs work.

Reply:
@naricc do you know?

Reply:
@safern Its just that with minijit/monointerpreter, there is no seperate product build (we only have one build that has both of them). Other runtime variants do have their own package, thus it needs to be parameterized by the runtimeVariant.